tokens/source/shadow/index.json

```diff
@@ -10,13 +10,33 @@
         "value": "0px 0px 0px 1px {color.alias.gray.20} inset"
       },
       "danger": {
-        "comment": "0px 0px 0px 1px red.60 inset",
-        "value": "0px 0px 0px 1px {color.alias.red.60} inset"
+        "_": {
+          "comment": "0px 0px 0px 1px red.60 inset",
+          "value": "0px 0px 0px 1px {color.alias.red.60} inset"
+        },
+        "strong": {
+          "comment": "0px 0px 0px 1px red.70 inset",
+          "value": "0px 0px 0px 1px {color.alias.red.70} inset"
+        }
       },
       "disabled": {
         "comment": "0px 0px 0px 1px gray.20 inset",
         "value": "0px 0px 0px 1px {color.alias.gray.20} inset"
       },
+      "filled": {
+        "danger": {
+          "comment": "shadow.border.danger but with 5px radius",
+          "value": "0px 0px 0px 5px {color.alias.red.60} inset"
+        },
+        "disabled": {
+          "comment": "shadow.border.disabled but with 5px radius",
+          "value": "0px 0px 0px 5px {color.alias.gray.20} inset"
+        },
+        "primary": {
+          "comment": "shadow.border.primary but with 5px radius",
```

Comment:
For the comments here, let's match the existing pattern instead of referencing a "like this token"

Reply:
Done